src/Umbraco.Core/Models/PublishedContent/VariationContextAccessorExtensions.cs

```diff
@@ -3,13 +3,32 @@
     public static class VariationContextAccessorExtensions
     {
         public static void ContextualizeVariation(this IVariationContextAccessor variationContextAccessor, ContentVariation variations, ref string culture, ref string segment)
```

Comment:
Does this method get used anymore? Else we should obsolete it

Reply:
It's still used in 2 locations in the `PublisedValueFallback` ([here](https://github.com/umbraco/Umbraco-CMS/blob/3eb4c642c39709840ce117a00a64c63f2526c7a4/src/Umbraco.Web/Models/PublishedContent/PublishedValueFallback.cs#L36) and [here](https://github.com/umbraco/Umbraco-CMS/blob/3eb4c642c39709840ce117a00a64c63f2526c7a4/src/Umbraco.Web/Models/PublishedContent/PublishedValueFallback.cs#L76)) as there does not seem to be a `contentId` available in those contexts. So I suppose in some fallback scenarios perhaps this new functionality will not be used yet.

Reply:
Ok no problemo we'll leave as-is for now